src/server.c

```diff
@@ -574,6 +574,54 @@ null_log_writer (GLogLevelFlags   log_level,
   return G_LOG_WRITER_HANDLED;
 }
 
+/* Bind server to available IPv4 and IPv6 local addresses
+ *
+ * The server must be NOT listening on any interface.
```

Comment:
What does this mean? Does this mean the server must not already be listening on any interface? If so, I suggest this comment:

  The server must NOT already be listening on any interface

Reply:
> Does this mean the server must not already be listening on any interface?

Exactly that. I will update the comment. Thanks for the suggestion!